lib/utils/utils.ts

```diff
@@ -75,9 +76,9 @@ export const isEmptyObject = (val: any): boolean => {
   return isObject(val) && Object.keys(val).length === 0;
 };
 
-/** Get the current date in the LocaleString format.
+/** Get the current date in the given dateFormat, if not provided formats with `DD/MM/YYYY HH:mm:ss.SSS`.
  */
-export const getTsString = (): string => (new Date()).toLocaleString();
+export const getTsString = (dateFormat?: string): string => moment().format(dateFormat ? dateFormat : 'YYYY-MM-DD hh:mm:ss.sss');
```

Comment:
the default format is a bit different from the one written in the comment.

also, you can write it as `dateFormat || 'YYYY...'` instead of the ternary operator.

Reply:
Good point !